eva_sub_cli/jinja_templates/project_details.html

```diff
@@ -0,0 +1,35 @@
+{% macro project_details_report(project_title, validation_date, submission_dir, vcf_fasta_analysis_mapping) -%}
+    <div class="report-section info">
+          {% if project_title %}
+             <p><b>Project Title:</b> {{ project_title }}</p>
+         {% else %}
+             <p><b>Project Title:</b></p>
```

Comment:
Maybe omit this or put "not found" if the title is missing (though I think it's required in the metadata so should probably be parsed out if possible)

Reply:
updated